Python/bltinmodule.c

```diff
@@ -2019,7 +2019,7 @@ builtin_min(PyObject *self, PyObject *const *args, Py_ssize_t nargs, PyObject *k
 }
 
 PyDoc_STRVAR(min_doc,
-"min(iterable, *[, default=obj, key=func]) -> value\n\
+"min(iterable, /, *[, default=obj, key=func]) -> value\n\
```

Comment:
```suggestion
"min(iterable, /, key=None)\n\
min(iterable, /, default, key=None)\n\
min(arg1, arg2, *args, /, *, key=None)\n\
```
just like sphinx docs